### DISCUSSION TITLE
HIP-00XX: Handshake Funding Proposal: Block Reward Diversion

### DISCUSSION THREAD

**Discussion by pcfreak30** [handshake-org/HIPs#65] [Polls]
## Background

This proposal is a variation on the original proposal to reallocate unclaimed airdrop and claim reward funds. The [original proposal](https://github.com/handshake-org/HIPs/discussions/64) identified approximately 1,078,754,316.63 HNS (about $15,106,020.07) in unused funds and suggested creating a community-managed multisig wallet to fund various Handshake ecosystem initiatives.

## New Funding Approach

While the original proposal aimed to use existing unclaimed funds, this variation proposes a different approach to source funding without altering the total HNS supply:

1.  **Block Reward Reduction**: Cut the current block reward in half.
2.  **Diversion to Multisig**: Divert the reduced portion of the block reward to the proposed community multisig wallet.
3.  **Temporary Measure**: Continue this diversion until the next halving event.

## Specifics

-   Next halving block: 340,000
-   Current block (at time of writing): ~235,126
-   Blocks until halving: ~104,874
-   Total HNS to be diverted: ~52,437,000 HNS (approximately 708,948.24 USD at current rates)

*These numbers will change based on the block picked for the hard fork*

## Rationale

1.  **Preserving Total Supply**: This approach doesn't alter the total HNS supply, maintaining the coin's economics.
2.  **Bootstrapping the Ecosystem**: Temporarily reduces miner rewards to bootstrap chain development and create a positive momentum.
3.  **Miner Assurance**: Miners can be confident their full rewards will resume after the next halving.

## Proposed Uses of Funds

The diverted funds could be used for various initiatives, as outlined in the original proposal:

-   Funding browser integrations (DANE, mobile wallets)
-   Developing non-Node.js Handshake nodes
-   Paying exchange listing fees
-   Writing full nodes in Go/Rust
-   Hosting community hackathons
-   Conducting security audits
-   Funding DEX integrations
-   Improving and evolving tools like Fingertip
-   Supporting core developers for maintenance and R&D

## Next Steps

1.  Community discussion and feedback on this variation
2.  If accepted, determine the exact block to begin the reward diversion
3.  Implement the technical changes necessary for the block reward split
4.  Set up the multisig wallet with the agreed-upon signers
5.  Establish a governance process for fund allocation

This proposal aims to kickstart funding for Handshake development while balancing the interests of miners and the broader ecosystem. 

We welcome community input and discussion on this approach.

It is also possible a combination of proposals are picked. *NOTHING* is set in stone.

## Timeline

This poll will be open for two weeks (until 2024.08.07).

**Comment by pcfreak30** [comment]
Tagging everyone involved in the original proposal @NetOpWibby @eskimo @nathanwoodburn @crymzyn @nodech @rithvikvibhu @falci @pinheadmz @PapoDisco

**Reply by agaamin** [reply]
Reducing miner incentive is not a good idea at this point. HNS is not worth a lot. I am pretty sure 50% reduction will leave them in the red.

**Reply by pcfreak30** [reply]
Can do less and possibly a combination of ideas. Point is it is an option to pull revenue from without changing supply.

**Comment by Nathanwoodburn** [comment]
The main issue I see with this proposal is the miners need incentives to mine. At the moment miners have very small margins so decreasing their profits could cause them to stop mining.

**Reply by pcfreak30** [reply]
That is why I had the idea to put it temp at the next halving. They are doing this knowing that the money is gradually going to fund the eco and thus increase the value long term. Any short term miners who don't care might quit, but if a majority of them are long term invested... they will see the upside and understand its for a few years only.

Plus there is also spec mining with asics and the potential of where the price might go.

TBH if we get this funding, the 1st thing *i think* should be done is fund a dex for liquidity and then we can see some legit BTC/HNS price action and enable more speculation/price discovery.

Also it is possible too we could do like 10% of the reward... but 50% just seems simpler.

**Comment by eskimo** [comment]
Yeah this can't work at all. It's barely any HNS and will make all miners unprofitable.

**Reply by pcfreak30** [reply]
` It's barely any HNS ` the total amount itself competes with the other approaches. I just view it an an option since many object inflation/changing scarcity.

**Reply by eskimo** [reply]
At the current value, it would take multiple months to generate even $100k which won't go very far. And again, that's $100k being taken from the miners.

**Reply by pcfreak30** [reply]
100k realistically could fund 1-2 major efforts depending on what they are. Hell, I would say the DEX would be the 1st effort.
